### ISSUE TITLE
estimate steepness with a prior

### ISSUE THREAD

**Issue by iantaylor-NOAA** [iantaylor-NOAA/Lingcod_2021#85]
As shown by @kellijohnson-NOAA in #83, estimating steepness with the prior used in the hake assessment showed relatively little change from the models with h fixed at 0.7. Estimating _h_ with no prior in earlier models caused estimates to hit the upper bound, so a prior is definitely useful but our rockfish prior doesn't apply, so the hake prior seems great.

The benefits of estimating steepness include
- better accounting for uncertainty in the stock-recruit relationship
- less risk of bias in M if steepness is misspecified (as shown in unpublished Castillo-Jordán et al. manuscript)
- avoids having to justify a fixed value

Risks include
- if the estimated values for the north and south models are very different, this would be harder to explain
- we will be asked to justify the choice of prior (probably with more scrutiny than given to the fixed value because it contains both a mean and a standard deviation, so twice as many numbers to question)

Here are some tasks related to this:
- [ ] Bring the hake prior into the existing scripted workflow for the future model runs (using code written by @kellijohnson-NOAA in eda9451f280cde3689a099b53d6f31493dbfb7bc) (Ian)
- [ ] Adapt text from the hake assessment description of the prior: https://github.com/pacific-hake/hake-assessment/blob/be6a0c53928dfc999fd42ed3d835d035eca964b9/doc/data-section/data.rnw#L593-L604
- [ ] Look at Myers et al. (1999) referenced by hake to make sure that there's not some better choice of prior for lingcod.

**Comment by iantaylor-NOAA** [comment]
Hey @james-thorson-NOAA, as a world-renowned expert on steepness priors for groundfish, do you have any thoughts on a prior we could use for lingcod?

**Comment by James-Thorson-NOAA** [comment]
Have you tried pulling the predictive distribution for FishLife?  

This would be done using

```R
library(FishLife)
out = Plot_taxa(Search_species(Genus="Ophiodon",Species="elongatus",add=FALSE)$match_taxonomy)
```

This then shows a predictive distribution with a mean of 0.67 and SD of 0.22.  This is obviously a very broad predictive distribution ... if you use it, perhaps explore a beta distribution approximation and please tell me if you want help!

**Comment by iantaylor-NOAA** [comment]
Thanks @James-Thorson-NOAA, for a quick and useful response!
I should have thought to look at FishLife.

A mean = 0.67 and SD = 0.22 doesn't work as a beta (at least with the parameterization used in Stock Synthesis), but mean = 0.65 and SD = 0.20 does work (calculations are [here](https://github.com/iantaylor-NOAA/Lingcod_2021/blob/main/unfit/plot_priors.R)). It's indeed very wide as shown in the figure below, so I'm not sure that it will provide enough information to keep steepness from wandering around too much. But it provides us with a useful option to explore. We could use the mean but reduce the SD.
![prior-steepness-comparisons](https://user-images.githubusercontent.com/4992918/123007559-3c82cb00-d36e-11eb-8c48-74cfb1442b39.png)

**Comment by James-Thorson-NOAA** [comment]
Well, I'm definitely curious to follow along as you investigate :) sorry
that I don't have more advice

**Comment by kellijohnson-NOAA** [comment]
Retros for the southern model are [here](https://drive.google.com/drive/folders/18zkEA-gaB9h0d20kAGgjNDG1jwLqkfUu?usp=sharing). No obvious degradation with estimating *h*.

**Comment by kellijohnson-NOAA** [comment]
[TOR](https://drive.google.com/file/d/10ZBWLzaPw8pxE0fwrocPll03fk4AMkd8/view?usp=drive_open) says `Use SSC-approved steepness prior for rockfish species in 2019 has a mean value of 0.72 and standard deviation of 0.16. If the assessment model does not estimate steepness, the STAT should fix the steepness value at 0.72.`

**Comment by iantaylor-NOAA** [comment]
Text on prior is in model-parameters.Rmd (just revised to make clear that 0.72 in ToR is for rockfish), and north vs south figures show prior relative to estimated distributions of steepness.
